app/assets/stylesheets/application/member-card.scss

```diff
@@ -86,4 +113,4 @@
       }
     }
   }
-}
+}
```

Comment:
You should see an icon like this 🚫, this indicates that this file doesn't have an extra line at the end of it. It's a good practice to have it